src/Ingredient.js

```diff
@@ -1,10 +1,44 @@
-import React from 'react';
-import IngredientContainer from './IngredientContainer.js'
-
-export default function Ingredient(props) {
-  return ( 
-    <h4>
-    {props.ingredient}
-    </h4>
-  )
-}
\ No newline at end of file
+import React, {Component} from 'react';
+import IngredientContainer from './IngredientContainer.js';
+import './Ingredient.css';
+
+export default class Ingredient extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isChosen: false
+    }
+  }
+
+  selectIngredient = () => {
```

Comment:
Thinking these two functions, selectIngredient and removeIngredient can be refactored into one function.

Reply:
I agree ⬆️

Reply:
done!